examples/connect-app/src/main.ts

```diff
@@ -1,36 +1,35 @@
-import { type DialOptions } from '@viamrobotics/rpc/src/dial';
 import * as VIAM from '@viamrobotics/sdk';
 
-async function connect(): Promise<VIAM.ViamClient> {
-  const credential = {
-    payload: '<API-KEY>',
-    type: 'api-key',
-  };
+const ORG_ID = import.meta.env.VITE_ORG_ID;
+const API_KEY_ID = import.meta.env.VITE_API_KEY_ID;
+const API_KEY_SECRET = import.meta.env.VITE_API_KEY_SECRET;
 
-  const dialOpts: DialOptions = {
-    authEntity: '<API-KEY-ID>',
-    credentials: credential,
+async function connect(): Promise<VIAM.ViamClient> {
+  const opts: VIAM.ViamClientOptions = {
+    credential: {
+      type: 'api-key',
+      authEntity: API_KEY_ID,
+      payload: API_KEY_SECRET,
+    },
   };
 
-  const client = new VIAM.ViamClient(dialOpts);
-  await client.connect();
+  const client = await VIAM.createViamClient(opts);
 
   return client;
 }
 
 const button = <HTMLButtonElement>document.getElementById('main-button');
 
 async function run(client: VIAM.ViamClient) {
-  // A filter is an optional tool to filter out which data comes back.
-  const opts: VIAM.FilterOptions = { componentType: 'camera' };
-  const filter = client.dataClient.createFilter(opts);
-
   try {
     button.disabled = true;
     const textElement = <HTMLParagraphElement>document.getElementById('text');
     textElement.innerHTML = 'waiting for data...';
 
-    const dataList = await client.dataClient.tabularDataByFilter(filter);
```

Comment:
This potentially returns a lot of data - updating to use `tabularDataBySQL` with a hard limit.